src/NosCore.GameObject/Helper/MinilandHelper.cs

```diff
@@ -253,19 +239,23 @@ private void LoadMinilandGifts()
                     }
                 }
             };
-        }
 
-        private void LoadMinilandMaxPoint()
+        public int[][] MinilandMaxPoint = new int[6][]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5262030)